.gitignore

```diff
@@ -1,5 +1,4 @@
 .DS_Store
 _site
-vendor/
-.Gemfile.lock
-.bundle/config
+
+Gemfile.lock
```

Comment:
Gemfile.lock should stay in the repo: https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore#4151540